lib/src/dashmon.dart

```diff
@@ -50,22 +50,18 @@ class Dashmon {
 
   void _processError(String line) {
     _print(line);
-    _process.kill();
-    exit(1);
+    // _process.kill();
+    // exit(1);
```

Comment:
```suggestion
```

Reply:
> Suggested change
>     // _process.kill();
>     // exit(1);

Oh, I didn't really understand how GitHub works, so I didn't realize that even the second commit was also included in the PR. This commit has nothing to do with issue #1, it is an experiment I did for myself. 
The purpose of this commit is to prevent dashmon from exiting when an error occurs during `flutter run`. For example, if I accidentally save a file with syntax errors and then hot reload runs, dashmon will exit, but it is tedious to restart it every time. So, I commented out those two lines. 
I've committed your suggestion, so please merge them if you  like!